### DISCUSSION TITLE
Native Mermaid support in NiceGUI (a Python library to create web-based interfaces)

### DISCUSSION THREAD

**Discussion by rodja** [mermaid-js/mermaid#4081] [Show and tell]
We fall in love with Mermaid on first sight. Thanks for creating such a great tool. It's super nice and powerful. Thats why we integrated it into [NiceGUI](https://nicegui.io):

```py
from nicegui import ui

ui.mermaid('''
graph LR;
    A --> B;
    A --> C;
''')

ui.run()
```

<img width="346" alt="Screen Shot 2023-02-11 at 05 44 21" src="https://user-images.githubusercontent.com/131391/218240789-7f5cce9f-ec06-4bf1-914c-20c2a040d891.png">

NiceGUI allows you create web based user interfaces within Python while still providing solid mechanisms for customizations with HTML/JS/CSS if needed. See this video for a show-case: https://youtu.be/HiNNe4Q32U4. We wrote the whole website https://nicegui.io/ with our own framework. You may also have a look at all the interactive [examples in the reference](https://nicegui.io/reference) and our [initial announcement on Reddit](https://www.reddit.com/r/Python/comments/10d6ugv/nicegui_let_any_browser_be_the_frontend_for_your/).

**Comment by sidharthv96** [comment]
Looks dope!
Can you add some links in our docs @ `/mermaid/packages/mermaid/src/docs/ecosystem`?

**Reply by rodja** [reply]
Sure, I've just created #4107.
